modules/openapi-generator/src/main/resources/dart-dio/api.mustache

```diff
@@ -77,7 +77,7 @@ class {{classname}} {
 
     _bodyData = {{#isMultipart}}FormData.fromMap({{/isMultipart}}<String, dynamic>{
       {{#formParams}}
-      {{^required}}{{^nullable}}if ({{{paramName}}} != null) {{/nullable}}{{/required}}r'{{{baseName}}}': {{#isFile}}MultipartFile.fromBytes({{{paramName}}}, filename: r'{{{baseName}}}'){{/isFile}}{{^isFile}}parameterToString(_serializers, {{{paramName}}}){{/isFile}},
+      {{^required}}{{^nullable}}if ({{{paramName}}} != null) {{/nullable}}{{/required}}r'{{{baseName}}}': {{#isFile}}MultipartFile.fromBytes({{{paramName}}}, filename: r'{{{baseName}}}'){{/isFile}}{{^isFile}}encodeFormParameter(_serializers, {{{paramName}}}, const FullType({{^isContainer}}{{{baseType}}}{{/isContainer}}{{#isContainer}}Built{{#isMap}}Map{{/isMap}}{{#isArray}}{{#uniqueItems}}Set{{/uniqueItems}}{{^uniqueItems}}List{{/uniqueItems}}{{/isArray}}, [{{#isMap}}FullType(String), {{/isMap}}FullType({{{baseType}}})]{{/isContainer}})){{/isFile}},
```

Comment:
would you sacrifice formatting of generated code to make this easier to read? It's a question, not a request.

Reply:
I ask myself this a lot but useful diffing of the generated code kinda seems important.

Reply:
Do you commit your autogenerated code to codebase? I'm used to generating it to untracked directories and generating everything in build pipelines. I do it in Java, TS and Dart.

Reply:
I do commit it for libraries but not for Flutter apps.
But what I actually mean are the changes in the generated example code in the PRs in this repository.

Reply:
Like the diff for the API files in this PR, they are pretty good to read.